packages/android_alarm_manager/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Remove support for the V1 Android embedding.
 * Updated Android lint settings.
 * Removed `-Werror` in Android builds.
+* Updated Android compileSdkVersion to 31.
```

Comment:
looks like present tense is preferred in changelogs. https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style